stl/inc/type_traits

```diff
@@ -1804,6 +1804,36 @@ inline constexpr bool is_nothrow_invocable_r_v =
     _Select_invoke_traits<_Callable, _Args...>::template _Is_nothrow_invocable_r<_Rx>::value;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+// STRUCT TEMPLATE is_layout_compatible
+template <class _Ty1, class _Ty2>
+struct is_layout_compatible : bool_constant<__builtin_is_layout_compatible(_Ty1, _Ty2)> {};
+
+template <class _Ty1, class _Ty2>
+inline constexpr bool is_layout_compatible_v = __builtin_is_layout_compatible(_Ty1, _Ty2);
+
+// STRUCT TEMPLATE is_pointer_interconvertible_base_of
+template <class _Base, class _Derived>
+struct is_pointer_interconvertible_base_of
+    : bool_constant<__builtin_is_pointer_interconvertible_base_of(_Base, _Derived)> {};
+
+template <class _Base, class _Derived>
+inline constexpr bool is_pointer_interconvertible_base_of_v = __builtin_is_pointer_interconvertible_base_of(
+    _Base, _Derived);
+
+// STRUCT TEMPLATE is_pointer_interconvertible_with_class

```

Comment:
This is a `FUNCTION TEMPLATE`. Ditto below.

Reply:
Fixed